sensu/plugins/check-dir-new-files.rb

```diff
@@ -0,0 +1,98 @@
+#! /usr/bin/env ruby
+#
+#   check-dir-new-files
+#
+# DESCRIPTION:
+#   Checks the number of specific files in a directory
+#
+# OUTPUT:
+#   plain text
+#
+# PLATFORMS:
+#   Linux, BSD
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#
+# USAGE:
+#   #YELLOW
+#
+# NOTES:
+#
+# LICENSE:
+#   Copyright 2014 Sonian, Inc. and contributors. <support@sensuapp.org>
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'fileutils'
+require 'time'
+
+class DirCount < Sensu::Plugin::Check::CLI
+  BASE_DIR = '/var/cache/check-dir-new-files'
```

Comment:
maybe we can use `/var/cache/sensu/check-dir-new-files` ?  just so its a little more obvious.  plus we can chown that dir in ansible so that sensu user can write to it and not need to sudo.